Documentation/microsoft-team.md

```diff
@@ -4,22 +4,39 @@ If you work on or with the .NET Team, you will need to onboard into various GitH
 
 ## Join .NET teams in dotnet and Microsoft orgs
 
-You need to link your GitHub and @microsoft.com accounts. Click the link:
+[Link your GitHub account](https://repos.opensource.microsoft.com/link) with Microsoft and then join our teams (in two organizations):
 
-* [Link your GitHub account](https://repos.opensource.microsoft.com/link)
+1. [Join the microsoft/dotnet team](https://repos.opensource.microsoft.com/Microsoft/teams/dotnet/join/)
+1. [Join the dotnet/microsoft team](https://repos.opensource.microsoft.com/dotnet/teams/microsoft/join/)
 
-You need to join teams in two organizations. Click the two links:
+## Security best practices
+
+Bad actors try to break into our accounts all the time (see ["failed login attempts" on your account](https://github.com/settings/security-log?q=action%3Auser.failed_login)). You need to apply the following guidance to (A) stay secure, and (B) maintain access to your account.
+
+The following best practices are required for org owners, and recommended for repo admins.
+
+* Do not use [SMS](https://en.wikipedia.org/wiki/SIM_swap_scam) for 2FA or as a recovery fallback (disable those options).
+* Do use the [GitHub mobile app](https://github.blog/2022-01-25-secure-your-github-account-github-mobile-2fa/) for 2FA.
```

Comment:
I would reword this as something like:

Register at least two two-factor authentication methods of the folllowing
- [GitHub mobile app](https://github.blog/2022-01-25-secure-your-github-account-github-mobile-2fa/)
- Register a [security key(s)](https://www.yubico.com/works-with-yubikey/catalog/github/)
- An authenticator app 

I have an authenticator app and Yubikey, but I don't use GitHub mobile.

Reply:
Good call.

Reply:
https://github.com/dotnet/core/pull/7418/commits/2b9ef058be30833f5a01ea4f1088f27607bb8426